### PR TITLE
Update AboutUs.md

### LINKED ISSUE #3: Update AboutUs page from default AB3 to CampusConnect
The AboutUs page needs to be updated from the default AboutUs from AB3 to one that's relevant to CampusConnect.

### PR DESCRIPTION
**Pull request by conradsoon**
Fixes #3

### CODE REVIEW

File: docs/AboutUs.md [hunk 1]
```diff
@@ -11,51 +11,47 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
...
+<img src="images/lordidiot.png" width="200px">
 
-[[homepage](http://www.comp.nus.edu.sg/~damithch)]
-[[github](https://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
+[[github](https://github.com/lordidiot)]
+[[portfolio](team/lordidiot.md)]
 
-* Role: Project Advisor
+- Role: Team lead
```
Comment (lordidiot) [inline docs/AboutUs.md]: Please add `Scheduling and tracking` as well

**Review by lordidiot — CHANGES_REQUESTED** [review]
One minor change.
Also, should we add surnames for clarity? Doesn't have to be full names but maybe surnames would be good.

File: docs/AboutUs.md (hunk 1 above)
Comment (conradsoon) [reply docs/AboutUs.md]: Have rectified the issues, please review again.

**Review by kokrui — APPROVED** [review]
LGTM, once this is merged to master I should be able to debug #5 as well

**Review by lordidiot — APPROVED** [review]
LGTM! Thanks for the quick fixes.

**Pull request merged**